mcux/mcux-sdk/components/conn_fwloader/driver_conn_fwloader.cmake

```diff
@@ -1,16 +1,30 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2023-24 NXP.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#-------------------------------------------------------------------------------
+
 #Description: Driver conn_fwloader; user_visible: True
 include_guard(GLOBAL)
 message("driver_conn_fwloader component is included.")
 
+if(NOT CONFIG_BUILD_WITH_TFM)
```

Comment:
Have we submitted this change to SDK team

Reply:
will do once we have the full list